generator/main.ts

```diff
@@ -33,19 +33,32 @@ async function generateInterface(
     file.read(path.resolve(TEMPLATE_DIR, 'interface_header.njk')),
     file.read(path.resolve(TEMPLATE_DIR, 'interface_cpp.njk'))
   ]);
+
   const idl_name = input_idl_path.replace('.idl', '').replace('.', '');
-  const header_file_path = path.resolve(output_path, idl_name + '_bridge.h');
-  const cpp_file_path = path.resolve(output_path, idl_name + '_bridge.cc');
+  for (let definition of idl_fragments.definitions) {
+    if (definition.type == idls.DefinitionType.Interface) {
+      const interfaceImpl: idls.InterfaceImpl =
+          definition as idls.InterfaceImpl;
 
-  // FIXME: each files can have one more interfaces. and this definition should
-  // be distinguished it is interface or not.
-  return Promise.all([
-    file.write(
-        header_file_path,
-        env.renderString(header_tmpl, idl_fragments.definitions[0])),
-    file.write(
-        cpp_file_path, env.renderString(cpp_tmpl, idl_fragments.definitions[0]))
-  ]);
+      // FIXME(Hwanseung) : when inferface name is CamelCase, should be change
```

Comment:
so actually i wanted to make class name to "TernaryCalculator" instead of "Ternary".
but i used class name to "Ternary" in order to void this issue.
i will resolve this issue at next patch and i am considering to use snake-case module (https://www.npmjs.com/package/snake-case)